models/marts/dag/fct_model_fanout.sql

```diff
@@ -19,15 +19,28 @@ models_without_children as (
 model_fanout as (
     select 
         all_dag_relationships.parent,
-        {{ dbt.listagg(measure='all_dag_relationships.child', delimiter_text="', '", order_by_clause='order by all_dag_relationships.child') }} as leaf_children
+        all_dag_relationships.child
     from all_dag_relationships
     inner join models_without_children
         on all_dag_relationships.child = models_without_children.parent
     where all_dag_relationships.distance = 1 and all_dag_relationships.child_resource_type = 'model'
+    group by 1, 2
+    order by 1, 2
```

Comment:
Can we just add a comment to all of these `order by`s explaining that they're necessary for the following `listagg` function

Reply:
Can do